docker/Dockerfile.frontend

```diff
@@ -1,16 +1,21 @@
-FROM node:21.5.0-slim
-
+FROM node:21.7.3-alpine3.20 AS builder
+RUN 
 WORKDIR /app
-
-COPY /frontend/package*.json .
-
-RUN npm i
-
+COPY ./frontend/package*.json .
+RUN npm ci --cache /tmp/empty-cache && \
+    npm cache clean --force && \
+    rm -rf /tmp/empty-cache
```

Comment:
Nit: I think we can use buildkit here to mount the cache in order to have faster builds

Reply:
I don't think it's necessary to manually manage the npm cache when using Buildkit. Buildkit automatically leverages layer caching for node modules based on your package-lock.json or yarn.lock file